src/App.tsx

```diff
@@ -1,61 +1,145 @@
 import './App.scss';
+import 'bulma/css/bulma.css';
+import { FormEvent, useState } from 'react';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { User } from './types/User';
+import { Todo } from './types/Todo';
+
+import { TodoList } from './components/TodoList';
+
+function getUserById(userId: number): User | null {
```

Comment:
Naming is important if you write `get`, you should get this user 100% or throw an error, in case of find you can return the user or null.
```suggestion
function findUserById(userId: number): User | null {
```